src/lib/components/Editor.svelte

```diff
@@ -72,19 +74,26 @@
 
 	$: extensions = get_extensions($editor_config);
 
-	function read_current_tab(current_tab: string) {
+	function read_current_tab(current_tab: string, is_image: boolean) {
 		if (!current_tab) return;
-		webcontainer.read_file(current_tab).then((file) => {
-			code = file;
+		if (!is_image) {
```

Comment:
can we inverse this boolean? get the special case out of the way first so the bottom has the regular case

feels better to me, but is **not** important

Reply:
sure